### PR TITLE
Correctly detect OS for non-containerized jobs

### LINKED ISSUE #90: Non-containerized jobs have 'ubuntu_22_04', should have 'ubuntu_20_04'
Reported by @tsafin

A test which exclusively fails on release_asan_clang11 always has os == 'ubuntu_22_04', which is wrong.
Job release_asan_clang11 runs right on the runner and not in a container. The runner has Ubuntu 20.04.

### PR DESCRIPTION
**Pull request by NickVolynkin**
Jobs which run on Linux runners without a container should have os == 'ubuntu_20_04', which is the current default OS for our GitHub runenrs.

When multivac fails to detect OS, the record will have os == 'unknown'. Having no default value will help us detect further errors.

Fixes #90

### CODE REVIEW

File: multivac/gather_data.py [hunk 1]
```diff
@@ -105,6 +106,16 @@ def github_time_to_unix(time: str) -> float:
     return time_to_unix
 
 
+# https://github.com/tarantool/tarantool/tree/master/.github/workflows
+os_matcher = re.compile(r"[a-z]+_[0-9]+(_[0-9]+)?")
+freebsd_matcher = re.compile(r"freebsd-[0-9]{2}")
+"""Jobs running right on the runner, not in a container"""
```
Comment (ylobankov) [inline multivac/gather_data.py:112]: I think it will be ok if it is a normal Python comment.

File: multivac/gather_data.py [hunk 2]
```diff
@@ -105,6 +106,16 @@ def github_time_to_unix(time: str) -> float:
     return time_to_unix
 
 
+# https://github.com/tarantool/tarantool/tree/master/.github/workflows
+os_matcher = re.compile(r"[a-z]+_[0-9]+(_[0-9]+)?")
```
Comment (ylobankov) [inline multivac/gather_data.py:110]: Usually, such constants should be in uppercase.

File: multivac/gather_data.py (hunk 1 above)
Comment (NickVolynkin) [reply multivac/gather_data.py]: fixed

File: multivac/gather_data.py (hunk 2 above)
Comment (NickVolynkin) [reply multivac/gather_data.py]: fixed

File: multivac/gather_data.py [hunk 3]
```diff
@@ -105,6 +106,16 @@ def github_time_to_unix(time: str) -> float:
     return time_to_unix
 
 
+# https://github.com/tarantool/tarantool/tree/master/.github/workflows
+OS_MATCHER = re.compile(r"[a-z]+_[0-9]+(_[0-9]+)?")
+FREEBSD_MATCHER = re.compile(r"freebsd-[0-9]{2}")
+# Jobs running right on the runner, not in a container
+NON_CONTAINERIZED_WORKFLOWS = re.compile(
```
Comment (ylobankov) [inline multivac/gather_data.py:113]: OTHER_WORKFLOWS_MATCHER?

**Review by ylobankov — APPROVED** [review]

File: multivac/gather_data.py (hunk 3 above)
Comment (NickVolynkin) [reply multivac/gather_data.py:113]: Not just any other workflows, but ones running on default runners. Maybe, DEFAULT_RUNNERS_MATCHER?

Comment (ylobankov) [reply multivac/gather_data.py]: Works for me.

**Pull request merged**